### PR TITLE
Explorer gasmasks can be equipped through their hoods

### PR DESCRIPTION
**Pull request by Anewbe**

### CODE REVIEW

**Comment by MisterLayne** [comment]
Will this mess with anything technical- Like how the hood is affected by weather, or armor, or anything like that?

**Comment by Anewbe** [comment]
Has no effect on the hood proper. Gasmask will be very marginally less useful, but it will be rewritten soon.

File: code/modules/clothing/masks/gasmask.dm [hunk 1]
```diff
@@ -73,7 +73,6 @@
 	icon_state = "explorer"
 	item_state_slots = list(slot_r_hand_str = "gas", slot_l_hand_str = "gas")
 	armor = list(melee = 10, bullet = 5, laser = 5,energy = 5, bomb = 0, bio = 50, rad = 0)
-	body_parts_covered = HEAD|FACE|EYES
```
Comment (Arokha) [inline code/modules/clothing/masks/gasmask.dm]: This will prevent this mask from protecting them from their eyes burning on phoron.

Comment (Neerti) [reply code/modules/clothing/masks/gasmask.dm]: Does it not inherit it from the base gasmask?

Comment (Arokha) [reply code/modules/clothing/masks/gasmask.dm]: Ouu. Good point. Wait, why was this defined to begin with then?

Comment (Anewbe) [reply code/modules/clothing/masks/gasmask.dm]: Probably stolen directly from TG?

**Pull request merged**